kotlinx-coroutines-core/build.gradle

```diff
@@ -223,6 +223,7 @@ task jvmStressTest(type: Test, dependsOn: compileTestKotlinJvm) {
     systemProperty 'kotlinx.coroutines.scheduler.keep.alive.sec', '100000' // any unpark problem hangs test
     systemProperty 'kotlinx.coroutines.semaphore.segmentSize', '2'
     systemProperty 'kotlinx.coroutines.semaphore.maxSpinCycles', '10'
+    systemProperty 'kotlinx.coroutines.bufferedChannel.segmentSize', '2'
```

Comment:
Please drop a comment on why exactly this is required. 
Nobody will remember that two years since now :)